source/browseMode.py

```diff
@@ -2107,10 +2107,10 @@ def _handleScrollTo(
 	) -> bool:
 		"""Handle scrolling the browseMode document to a given object in response to an event.
 		Subclasses should call this from an event which indicates that the document has scrolled.
-		@postcondition: The virtual caret is moved to L{obj} and the buffer content for L{obj} is reported.
-		@param obj: The object to which the document should scroll.
-		@return: C{True} if the document was scrolled, C{False} if not.
-		@note: If C{False} is returned, calling events should probably call their nextHandler.
+		- postcondition: The virtual caret is moved to L{obj} and the buffer content for L{obj} is reported.
+		:param obj: The object to which the document should scroll.
+		:return: True if the document was scrolled, False if not.
+		- note: If False is returned, calling events should probably call their nextHandler.
```

Comment:
https://devguide.python.org/documentation/markup/

```suggestion
		- postcondition: The virtual caret is moved to *obj* and the buffer content for *obj* is reported.
		:param obj: The object to which the document should scroll.
		:return: ``True`` if the document was scrolled, ``False`` if not.
		- note: If ``False`` is returned, calling events should probably call their ``nextHandler``.
```